src/main/scala/gitbucket/core/plugin/PluginRegistry.scala

```diff
@@ -409,6 +409,13 @@ object PluginRegistry {
     }
   }
 
+  def getPluginInfoFromClassLoader(classLoader: ClassLoader): Option[PluginInfo] = {
```

Comment:
Using ClassLoader as key of PluginInfo is an idea. Ideally, we might have to make possible to provide a general way to get PluginInfo by the plugin id in the future. But it would require large refactoring across the plugin system.

Anyway, `getPluginInfoFromClassLoader` should not be at a general place as a public API. I think it should be implemented as a private or protected method in `PluginAssetsServlet`.

Reply:
> it should be implemented as a private or protected method in `PluginAssetsServlet`.

Can I get `instance` from `PluginAssetsServlet`?

Reply:
I mean this method shouln't be a public API. Being package private or any other solutions are acceptable. Don't you have good idea?

Reply:
I see. My idea is moving plugin related servlets to under `plugin` package, but your choise looks better.